<launch>
  <group ns="mes">
    <node pkg="rsd_mes_client" name="mes_client" type="rsd_mes_sorting_client_node.py" output="screen" >
	<param name="mes_command_topic" value="command" />
	<param name="mes_status_topic" value="status" />
	<param name="mes_incoming_topic" value="incoming" />
	<param name="mes_outgoing_topic" value="outgoing" />
	<param name="mes_order_topic" value="picklist" />
	<!--param name="hi_topic_udt" value="hi_topic" /-->
	<param name="update_duration" value="1" /> <!-- Unit: Seconds -->
	<param name="server_address" value="http://192.168.1.50:8000" />
	<param name="robot_id" value="3" />
    </node>
  </group>
</launch>
